pkg/observability/container_logs_service.go

```diff
@@ -6,7 +6,8 @@ import (
 )
 
 const (
-	containerLogsTemplate = "/api/{spaceId}/observability/logs/sessions/{sessionId}"
+	containerLogsTemplate      = "/api/{spaceId}/observability/logs/sessions/{sessionId}"
+	beginContainerLogsTemplate = "/api/{spaceId}/observability/logs/sessions"
```

Comment:
A small thing, but you can use a syntax to combine this template into one like Liam's done here: https://github.com/OctopusDeploy/go-octopusdeploy/pull/348/files#diff-b6f19a2ed75e8c6ba9b64d2d77ae5d5497aa7d3d0a0e3a217a7709705d105ef9R8

```suggestion
	containerLogsTemplate      = "/api/{spaceId}/observability/logs/sessions{/sessionId}"
```

Reply:
updated now in [892a1a6](https://github.com/OctopusDeploy/go-octopusdeploy/pull/347/commits/892a1a6cb976dabf258a816ed9dde9c6fde4f7f8)